model_utils/tracker.py

```diff
@@ -79,16 +80,34 @@ def __get__(field, instance, owner):
                     self.saved_data[field.field_name] = deepcopy(value)
                 return data[field.field_name]
 
+        class FileDescriptorTracker(FileDescriptor):
```

Comment:
I don't think so. The internals of `DeferredAttribute.__get__` and `FileDescriptor.__get__` are different. For example, for `FileDescriptor` doesn't need a check

    if data.get(field.field_name, field) is field